tools/deps_info.py

```diff
@@ -186,6 +186,7 @@
   'wgpuBufferGetMappedRange': ['malloc', 'free'],
   'wgpuBufferGetConstMappedRange': ['malloc', 'free'],
   'emscripten_glGetString': ['malloc'],
+  '$stringToNewUTF8': ['malloc']
```

Comment:
This isn't quite how deps_info works. This file is used before JS is linked. What it solves is this situation:

* C function `foo_bar` calls JS function `someJS`
* `someJS` calls C function `baz_baz`

Then deps info sees, by looking at the object files - before JS - that `foo_bar` is linked in. A key in `deps_info` for `foo_bar` will tell it that `baz_baz` is needed, even if it can't see that directly.

So the keys in deps_info are C functions, not JS ones.